packit/api.py

```diff
@@ -554,21 +554,25 @@ def create_srpm(
             logger.error(f"Failed when getting Bodhi updates: {exc}")
             return []
 
+    @staticmethod
+    async def status_main(status):
+        """
+        Schedule repository data retrieval calls concurrently.
+        :param status:
+        :return: awaitable tasks
+        """
+        res = await asyncio.gather(
+            PackitAPI.status_get_downstream_prs(status),
+            PackitAPI.status_get_dg_versions(status),
+            PackitAPI.status_get_up_releases(status),
+            PackitAPI.status_get_builds(status),
+            PackitAPI.status_get_updates(status),
+        )
+        return res
+
     def status(self):
         status = Status(self.config, self.package_config, self.up, self.dg)
-        loop = asyncio.get_event_loop()
-        try:
-            res = loop.run_until_complete(
-                asyncio.gather(
-                    self.status_get_downstream_prs(status),
-                    self.status_get_dg_versions(status),
-                    self.status_get_up_releases(status),
-                    self.status_get_builds(status),
-                    self.status_get_updates(status),
-                )
-            )
-        finally:
-            loop.close()
+        res = asyncio.run(self.status_main(status))
```

Comment:
[asyncio.run()](https://docs.python.org/3/library/asyncio-task.html#asyncio.run) is new in Python 3.7 and we claim to [support also 3.6](https://github.com/packit-service/packit/blob/master/setup.cfg#L18).

Do you think it would be possible to fix it so that we can still support the 3.6? (3.6 is default in RHEL/CentOS 8)

Reply:
@jpopelka fair point. I have added backward compatibility code within a version check. Two reasons for doing that:
1. We will use the recommended call `async.run` for Python 3.7.
2. At a later point, when we do not have to support Python 3.6, we will be able to safely remove that block of code.